boards/stm32f412gdiscovery/Makefile

```diff
@@ -13,13 +17,14 @@ OPENOCD_OPTIONS=-f openocd.cfg
 install: flash
 
 .PHONY: flash-debug
-flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
+flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
```

Comment:
This is just identical to the flash target now, right? Shouldn't this still be debug?

Reply:
All this night testing of boards takes its toll, sorry about this.